src/apiflask/scaffold.py

```diff
@@ -338,6 +339,7 @@ def output(
         examples: t.Optional[t.Dict[str, t.Any]] = None,
         links: t.Optional[t.Dict[str, t.Any]] = None,
         content_type: t.Optional[str] = 'application/json',
+        headers: t.Optional[t.Dict[str, Field]] = None,
```

Comment:
We only need to accept schema object/instance type. You can refer to the implementation of [APIFairy](https://github.com/miguelgrinberg/APIFairy/blob/main/src/apifairy/core.py#L282-L289).

Reply:
Done it!